oat-sa/extension-tao-itemqti
# Hotfix/AUT-937/font selector wrong value

Related to: https://oat-sa.atlassian.net/browse/AUT-937
Backport of 2 commits from https://github.com/oat-sa/extension-tao-itemqti/pull/1862

When user create couple of items with different font family was issue that internal variable customStylesheet was not reseted on init and for next item in Authoring StyleEditor has customStylesheet !==null and don't add stylesheet to item.
As a result item XML was without <stylesheet> tag.

How to test:

1. Create item, set Font family, save
2. Repeat step 1, to have 3 items with different fonts
3. Create test with these items
4. Preview test or create delivery and check that font families are applied
